src/etc/inc/filter.inc

```diff
@@ -3601,6 +3602,7 @@ EOD;
 			$ipfrules .= "pass out {$log['pass']} route-to ( {$ifcfg['if']} {$gw} ) from {$ifcfg['ip']} to !{$ifcfg['sa']}/{$ifcfg['sn']} tracker {$increment_tracker($tracker)} keep state allow-opts label \"let out anything from firewall host itself\"\n";
 			if (is_array($ifcfg['vips'])) {
 				foreach ($ifcfg['vips'] as $vip) {
+                    if ($vip['mode'] == "proxyarp") { continue; } 
```

Comment:
Please use [K&R, BSD KNF variant style](https://doc.pfsense.org/index.php/Developer_Style_Guide#Indent_style), even for simple statements, to keep them consistent and easy to read, and be careful to use matching and proper indentation.